lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -1041,14 +1048,16 @@ void LoopNetworkRewriter::buildExitNetwork(
 }
 
 LogicalResult LoopNetworkRewriter::processOuterLoop(Location loc,
-                                                    LoopInfo &loopInfo) {
+                                                    CFGLoop *loop) {
   // We determine the exit pairs of the loop; this is the in-loop nodes
   // which branch off to the exit nodes.
   llvm::SmallSet<ExitPair, 2> exitPairs;
-  for (auto *exitNode : loopInfo.exitBlocks) {
+  SmallVector<Block *> exitBlocks;
+  loop->getExitBlocks(exitBlocks);
+  for (auto *exitNode : exitBlocks) {
```

Comment:
nit: unnecessary brackets around outer for loop

Reply:
I'm not a big fan of dropping brackets around a multi-line statement, tbh.